### ISSUE TITLE
Upgrade eemeter (and pandas) dependencies

### ISSUE THREAD

**Issue by terencebarrett** [EPAENERGYSTAR/epathermostat#6]
A project (soon to be open-sourced) I am working on for VEIC would like to depend on this one (currently using eemeter 0.4.8) as well as the latest eemeter release (1.3.3), and so an upgrade of this project's dependencies would make this possible in our and others' projects.

I intend to do this work (including upgrading the pandas dependency to latest) and will hopefully have a pull request open for it soon.

**Comment by terencebarrett** [comment]
This work is progressing - the thermostat code has been updated and passes all tests with the latest version of eemeter (and pandas). Now I am updating the notebooks. The notebooks tutorial.ipynb and test_data_generation.ipynb have been updated and produce correct results with the example thermostat files provided. 

However, the last notebook, thermostat_data_generation.ipynb, is giving me some trouble. After updating, it generates thermostats, but these thermostats raise errors in the tutorial notebook. To debug this I attempted to run the released version of the notebook with the old eemeter but it does not run, for a peculiar reason. The pip-installed eemeter 0.4.8 includes a number of files in the module root (not seen in that release's branch on GitHub) that provide different versions of code that are also in the module directories. The notebook imports from the module directories but attempts to use code that is from the files in the module root. An example is the class ISDWeatherSource -- the notebook is looking to use a method on it called hourly_temperatures(), which is in the module-root code, but not in the module-directory code that was imported. Before I continue trying to sort this out, I wanted to check in to make sure this example-thermostat-generating notebook is still needed going forward - perhaps the existing examples will suffice?
